packages/react-components/src/components/ChatMessage/MessageComponents/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -135,7 +135,11 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
   const isBlockedMessage =
     false || /* @conditional-compile-remove(data-loss-prevention) */ message.messageType === 'blocked';
   const chatMessageCommonStyles = useChatMessageCommonStyles();
-
+  /* @conditional-compile-remove(attachment-upload) */
+  const hasMultipleAttachments = useMemo(() => {
+    const length = (message as ChatMessage).attachments?.length ?? 0;
```

Comment:
Can we reuse `doesMessageContainMultipleAttachments` here too?

Reply:
thanks updated